index.html

```diff
@@ -12,40 +12,289 @@
+                            <p class="wprk-description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                                industry's standard
+                            </p>
+                            <div class="work-tech-stack">
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
+                    </div>
+                </div>
+                <div class="card-container">
+                    <div class="work-card">
+                        <div class="work-card-body">
+                            <h1>Professional Art print Data</h1>
+                            <p class="wprk-description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                                industry's standard
+                            </p>
+                            <div class="work-tech-stack">
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- About me -->
+        <section id="about-me" class="about-me ">
+            <div class="intro">
+                <h1>About me</h1>
+                <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work
+                    and
+                    experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button class="btn-get-my-resume" type="button">Get my resume</button>
```

Comment:
- [ ] Kindly remove border from the button so it looks like the button in the Figma design